app/src/main/res/layout/item_popup_row.xml

```diff
@@ -71,5 +71,19 @@
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@+id/popupField"
                 app:layout_constraintVertical_bias="0.975" />
+
+        <Spinner
+                android:id="@+id/cvdValues"
```

Comment:
Similar to the comments above for property names, I suggest to give UI elements an id that indicates the type of UI element. I.e. in this case I suggest: 

```suggestion
                android:id="@+id/codedValueDomainSpinner"
```

Reply:
updated